src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -1894,7 +1890,7 @@ private static IExternalSerializer GetFallbackSerializer(bool useJsonSerializer)
 #if NETSTANDARD
                 throw new OrleansException("Can't use binary formatter as fallback serializer while running on .Net Core");
 #else
-                serializer = new BinaryFormatterSerializer();
+                serializer = new IlBasedFallbackSerializer(new IlBasedSerializers());
```

Comment:
Why is this not under `if NETSTANDARD`? Should we have a config flag for choosing `BinaryFormatterSerializer` vs. `IlBasedFallbackSerializer`, so that we can test the latter on full .NET?